search-commons/src/test/resources/mapping_test_tickets.json

```diff
@@ -16,14 +16,6 @@
       "type": "nested",
       "include_in_parent": true
     },
-    "organizations": {
-      "properties": {
-        "partOf": {
-          "type": "nested",
-          "include_in_parent": true
-        }
-      }
-    },
```

Comment:
this isn't used, `organizations ` should have been `organization`

but it doesn't matter , default mapping works well